resources/xrp.js

```diff
@@ -23,9 +23,24 @@ window.onload = () => {
     saveButton.onclick = (e) => {
         e.preventDefault();
         console.log("Save Button Clicked");
-    
         try {
-            JSON.parse(configJsonEntry.value);
+            console.log(configJsonEntry.value.toString());
+
+            // Check if can parse
+            let jsonObj = JSON.parse(configJsonEntry.value);
+            
+            // Check if password 8 characters or more
+            let password = jsonObj["network"]["defaultAP"]["password"];
```

Comment:
Should put a check around this to ensure that all the fields are indeed present

`if (jsonObj["network"] && jsonObj["network"]["defaultAP"]`...

Reply:
I'll add some more checks and throws. I figured it would throw an exception anyway, but it's better to be more explicit. New commit coming soon.